### PR TITLE
RnD Weapon Changes

### PR DESCRIPTION
**Pull request by GhostEcho**
Goodbye ballistics, you were boring, overpowered, and no one liked you besides powergamers
Adds: 
Tesla Revolver: A 6 shot energy pistol that does 52 damage on an unarmored person with one full chamber, arcs to nearby targets for reduced damage, be careful,  will arc to you too! Yellow gloves can prevent this from harming 
One Point Gravity Manipulator: Shoots a repulsive bolt that will throw things a short distance away from the impact point, try laying traps with spears and bolas! Slightly inaccurate
Temp Rifle: Makes people cold.

### CODE REVIEW

**Comment by improvedname** [comment]
Gravity gun + x objects = server suffer

**Comment by GhostEcho** [comment]
Itll be fiiiine, maybe

**Comment by improvedname** [comment]
Well lets see how vault will handle this new content

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -261,7 +261,7 @@
 	icon_state = "freezegun"
 	desc = "A gun that changes temperatures."
 	ammo_type = list(/obj/item/ammo_casing/energy/temp, /obj/item/ammo_casing/energy/temp/hot)
-	cell_type = "/obj/item/stock_parts/cell/high"
+	cell_type = /obj/item/stock_parts/cell/ammo/mfc
```
Comment (rhicora) [inline code/modules/projectiles/guns/energy/special.dm]: Does cell_type expect and object reference, or a string of an object reference?

Comment (improvedname) [reply code/modules/projectiles/guns/energy/special.dm]: Just remove the string parent has that as basic celltype

Comment (GhostEcho) [reply code/modules/projectiles/guns/energy/special.dm]: Just went with Hyenas, still works fine, suppose there's some cleanup opprotunities here tbh

**Pull request merged**